doc/api/crypto.md

```diff
@@ -2883,11 +2883,10 @@ const receivedPlaintext = decipher.update(ciphertext, null, 'utf8');
 
 try {
   decipher.final();
+  console.log(receivedPlaintext);
 } catch (err) {
   console.error('Authentication failed!');
```

Comment:
maybe add `err.reason` to the error message?

Reply:
Sorry about the delay, I missed this! `err.reason` is undefined in Node.js 12.1.0. Do you mean `err.message`?

Reply:
I assumed from the code that `ERR_get_error()` would return non-zero, after DecrypFinal_ex() returned failure, but apparently it is not (I didn't debug the source, but that's why .reason would not be set). So, ignore this comment (the err.message is a bit obscure, it probably doesn't need to be added).